login.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+"""
+Purpose: to serve as an example on how to query the login endpoint for session
+resuse
+
+Input: URL, cluster username, and password
+
+Returns: sessiondID cookie
+
+Note: Before running, provide the path to your certificate chain or set the
+request argument to verify=False. However, with this setting you maybe subject
+to a man-in-the-middle attack. Please only use for local
+development or testing
+"""
+
+import requests
+import json
+import base64
+
+#prompt for url and provide example
+print("Please provide the URL..")
+url = input("Example: https://cluster.domain.internal/rest/v1/login \n")
+
+#prompt for username and password
+username = input("Username: ")
+password = input("Password: ")
+
+#base64 encode and format for header
+encoded = str(base64.b64encode(bytes('{0}:{1}'.format(username, password), 'utf-8')), 'utf-8')
+auth = str.format('Basic ' + encoded)
+
+# request body and headers
+payload = json.dumps({
+  "username": username,
+  "password": password,
+  "useOIDC": False
+})
+headers = {
+  'Authorization': auth,
+  'Content-Type': 'application/json'
+}
+
+#login request
+response = requests.request("POST", url, headers=headers, data=payload,
+        verify='/path/to/certfile')
+
+print(response.text)
+
```

Comment:
This should probably include a logout as well I should think.

Reply:
I would add bonus points if that was implemented as a [context manager](https://docs.python.org/3.9/library/stdtypes.html#typecontextmanager). I would make sure to note that a login - do something - log out is sufficient.

Reply:
not sure who our audience is for this, a context manager might be too much.

Reply:
> I should think you would want to have something demonstrating a bit of how to use these end points. It doesn't need to be much, just answering "how do I use this response?"

Yeah, this is a good point. I should add more to answer that question which should also include a logout.

I like the context manager challenge. Thinking I should start with the above though.

Reply:
The context manager stuff, I thought about it and ended with while cool, it could distract from answering that question.